lib/plugins/plugin-validator.spec.js

```diff
@@ -43,6 +43,13 @@ describe('plugin-validator', () => {
   })
 
   describe('unexpected keys', () => {
+    afterEach(() => {
+      // prevent the process exitcode of 100 from failing the tests in the CI
```

Comment:
The comment describes the code but not the reason.  What does exit code 100 mean and why do we need to avoid that failing the CI?

Reply:
Please see here where the test fails in the CI: https://github.com/alphagov/govuk-prototype-kit/actions/runs/6668446182/job/18124028349

The tests actually pass, but the action fails due to the process completing with exit code 100

The exit code 100 is set in the code and now that we have moved to node 20, the unit tests fail when we don't unset it from 100

The fix in the tests was the only way I could get this to pass.